src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/RadioactivityTask.java

```diff
@@ -73,17 +73,19 @@ public void run() {
     }
 
     private void handleRadiation(@Nonnull Player p, @Nonnull PlayerProfile profile) {
-        if (p.getGameMode() == GameMode.CREATIVE ||
-                p.getGameMode() == GameMode.SPECTATOR) return;
+        if (p.getGameMode() == GameMode.CREATIVE || p.getGameMode() == GameMode.SPECTATOR) {
+         return;
+         }
         Set<SlimefunItem> radioactiveItems = SlimefunPlugin.getRegistry().getRadioactiveItems();
 
         int exposureTotal = 0;
         UUID uuid = p.getUniqueId();
 
         if (!profile.hasFullProtectionAgainst(ProtectionType.RADIATION)) {
             for (ItemStack item : p.getInventory()) {
-                if (item == null ||
-                        item.getType() == Material.AIR) continue;
+                if (item == null || item.getType() == Material.AIR) {
+                    continue;
+                    }
```

Comment:
This line is still wrong, the indentation should be on the same level as the if condition.